dev/doc/changes.md

```diff
@@ -9,6 +9,14 @@
 
 - Renaming `LSet` into `Level.Set` and `LMap` into `Level.Map`
 
+### Concrete syntax
```

Comment:
The changelog entry should be created with `dev/tools/make-changelog.sh` rather than added directly here.

Reply:
These changes are about the OCaml API which don't go in the `doc/changelog` directory as far as I know.

Reply:
It would fit into the "misc" category.  If you don't want to put it there, maybe use the title "OCaml API" or "OCaml API (internals)" instead of "Concrete syntax".

Reply:
My point is that users who are not plugin developers do not need to know about the changes in this PR as they are without visible effect on using Coq.

On the other side, the related #14640 is normally going to introduce changes of behavior and I agree that it will need a "user-level" change log.